### ISSUE TITLE
Add Planning Worksheet to Project 3: platform game?

### ISSUE THREAD

**Issue by nathanielgranor** [TEALSK12/introduction-to-computer-science#132]
We link to a general Planning Worksheet in many projects and labs. Unit 2's Pong project even has a customized version of the planning worksheet specific to pong.

We should include the planning worksheet in the Project 3 descriptions (or even a customized version of it)

**Comment by aspiece** [comment]
This would be nice to have. It would be great to have someone create something like this for the labs in Markup. If anyone is interested, let me know. We can consider this on a later release.

**Comment by aspiece** [comment]
Added to the backlog for consideration

**Comment by j-mag-teals** [comment]
I actually did this two years ago for my (remote) Hillsboro students -- this is intended to be a living e-document (we used Google Classroom as our LMS), so there's no "space" to write in.  It could be adapted for printing and writing on.

[Generic Cloud Platform Game Planning Sheet.docx](https://github.com/TEALSK12/introduction-to-computer-science/files/4421071/Generic.Cloud.Platform.Game.Planning.Sheet.docx)
y students.
